### DISCUSSION TITLE
desktop-trampoline.exe detected as virus trojan.Kryplod.fd

### DISCUSSION THREAD

**Discussion by mawanirwansyah** [community/community#49577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi
I did a test on the desktop-trampoline.exe file and it read that there was an indication of a malware virus
Url : https://www.virustotal.com/gui/file/990de76de3c1d1e17eabac024dab4446c40eea0d854df022817c61f3418ff1f8/detection

I want to ask, is the virus detection dangerous and what is the solution for this?
![trojan](https://user-images.githubusercontent.com/64650892/223915265-4f3b1f7a-8d35-4d9e-abb9-0bcaa6e6e02a.png)

**Comment by chrissc6** [comment]
Any updates on this?
